battery-status/battery-interface-idlharness.https.html

```diff
@@ -7,71 +7,26 @@
 <script src="/resources/testharnessreport.js"></script>
 <script src="/resources/WebIDLParser.js"></script>
 <script src="/resources/idlharness.js"></script>
-<script type="text/plain" class="untested">
-interface Navigator { };
+<script>
+"use strict";
 
-interface EventTarget {
-  void addEventListener(DOMString type, EventListener? callback, optional boolean capture);
-  void removeEventListener(DOMString type, EventListener? callback, optional boolean capture);
-  boolean dispatchEvent(Event event);
-};
+promise_test(async () => {
+  const idl_array = new IdlArray();
+  const dom_idl = await fetch("/interfaces/dom.idl").then(r => r.text());
+  const battery_idl = await fetch("/interfaces/battery.idl").then(r => r.text());
+  const manager = await navigator.getBattery();
 
-[Callback]
-interface EventListener {
-  void handleEvent(Event event);
-};
-[TreatNonObjectAsNull]
-callback EventHandlerNonNull = any (Event event);
-typedef EventHandlerNonNull? EventHandler;
-</script>
-<script type="text/plain">
-partial interface Navigator {
-    Promise<BatteryManager> getBattery ();
-};
+  idl_array.add_untested_idls(dom_idl);
```

Comment:
@Honry - Might be best to use `add_untested_idls(dom_idl, { only: ['Foo', 'Bar'] })` as dom.idl is rather extensive.

Reply:
Yeah, we don't need to test anything from `dom_idl`, so we called `add_untested_idls`, i.e. `dom_idl` will be skipped by `test()`.
`only` is used in `add_tested_idls`,  to only allow specified member to be added to `test()`.

Reply:
Oh, you are right, `only` can also restrict members to be added to `add_untested_idls`.

Reply:
I tried to validate if `only` can be used to restrict members to be added to `add_untested_idls`, seems the answer is no. I set an irrelevant interface to `only`,  no change to test result.

Reply:
I added `only` support for `add_untested_idls` in https://github.com/w3c/web-platform-tests/pull/10100.

When you add irrelevant untested interfaces, the only time that it causes a problem is if `A implements B` and you add that as untested, but `B` is never defined. This is a common problem pulling in large IDL files, as some parts might depend on definitions in other files.

Reply:
Oh, I didn't realize this new change as I used an old branch. And it works now, will update the code soon.